.styleci.yml

```diff
@@ -0,0 +1,6 @@
+preset: psr2
+
+enabled:
+  - short_array_syntax
```

Comment:
I prefer short syntax too myself – PHP 5.3 isn't [too fond](http://php.net/manual/en/migration54.new-features.php) of it though 😜